### PR TITLE
Added file check to forms files. I think it can be taken a step furth…

### PR DESCRIPTION
**Pull request by teryhill**
…er at some point.

Added file check to forms files. I think it can be taken a step further
at some point.  This is basically the same code that is used in other
forms done by ZH. I just modified it to work as a function.

### CODE REVIEW

File: library/formsoptions.inc.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+/*
+ * Functions to be used in the forms.
```
Comment (aethelwulffe) [inline library/formsoptions.inc.php]: "clinical/visit/encounter forms?"

Comment (aethelwulffe) [inline library/formsoptions.inc.php]: And include a sentence of how and why.

Comment (teryhill) [inline library/formsoptions.inc.php]: I get the clinical/visit/encounter forms change but not sure about what you want here or do you mean this for the function itself?

Comment (aethelwulffe) [inline library/formsoptions.inc.php]: Noooo.  Nothing that deep.  Just suggesting a bigger explain'n of what the file does.

Comment (teryhill) [inline library/formsoptions.inc.php]: Did that explain it enough?

Comment (aethelwulffe) [inline library/formsoptions.inc.php]: I suppose so.  I was just suggestion of "stuff for encounter forms, just common functions, don't put any BS in here or I will bomb.gzip your butt."
It is fine, and I guess I am just wasting time.  This stuff is ready to be merged...Oh Third set of Eyes! 👁‍🗨 👁

**Comment by aethelwulffe** [comment]
Real good, real clear, and this should get merged pronto IMHO

**Comment by tmccormi** [comment]
ok...

**Pull request merged**